From fa6cead3e65479fb9d71cc0862980f3bfd3d98c5 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Tue, 20 May 2014 13:30:38 +0800
Subject: [PATCH 1/3] iw/autopart_type.py: remove un-verified options

We only plan to support "use all spaces" and "Create Custom Layout" in
RCPL 7, so remove others.

Upstream-Status: Inappropriate [WR specific]

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 pyanaconda/iw/autopart_type.py | 20 --------------------
 1 file changed, 20 deletions(-)

diff --git a/pyanaconda/iw/autopart_type.py b/pyanaconda/iw/autopart_type.py
index b8eaf34..3f02f00 100644
--- a/pyanaconda/iw/autopart_type.py
+++ b/pyanaconda/iw/autopart_type.py
@@ -250,26 +250,6 @@ class PartitionTypeWindow(InstallWindow):
                                           "<b>Tip:</b> This option will remove "
                                           "data from the selected device(s).  Make "
                                           "sure you have backups."))
-        self.buttonGroup.addEntry("replace", _("Replace Existing _Linux System(s)"),
-                                  pixmap=gui.readImageFromFile("partscheme-replace.png"),
-                                  descr=_("Removes all Linux partitions on the "
-                                          "selected device(s). This does "
-                                          "not remove other partitions you may have "
-                                          "on your storage device(s) (such as VFAT or "
-                                          "FAT32).\n\n"
-                                          "<b>Tip:</b> This option will remove "
-                                          "data from the selected device(s).  Make "
-                                          "sure you have backups."))
-        self.buttonGroup.addEntry("shrink", _("_Shrink Current System"),
-                                  pixmap=gui.readImageFromFile("partscheme-shrink.png"),
-                                  descr=_("Shrinks existing partitions to create free "
-                                          "space for the default layout."))
-        self.buttonGroup.addEntry("freespace", _("Use _Free Space"),
-                                  pixmap=gui.readImageFromFile("partscheme-freespace.png"),
-                                  descr=_("Retains your current data and partitions and "
-                                          "uses only the unpartitioned space on the "
-                                          "selected device(s), assuming you have enough "
-                                          "free space available."))
         self.buttonGroup.addEntry("custom", _("Create _Custom Layout"),
                                   pixmap=gui.readImageFromFile("partscheme-custom.png"),
                                   descr=_("Manually create your own custom layout on "
-- 
1.8.3.1

